providers/sendinblue/src/lib/brevo.provider.ts

```diff
@@ -27,17 +29,16 @@ export class SendinblueEmailProvider implements IEmailProvider {
       senderName: string;
     }
   ) {
-    this.transactionalEmailsApi = new TransactionalEmailsApi();
-    this.transactionalEmailsApi.setApiKey(
-      TransactionalEmailsApiApiKeys.apiKey,
-      this.config.apiKey
-    );
+    this.defaultClient = brevo.ApiClient.instance;
```

Comment:
I'm a bit concerned that this will be reused across class institutions. Can we ensure that we have a brand new object here each time instead of referencing this like it? @djabarovgeorge wdyt?

Reply:
Agree, we should have same behavior between the providers, i am bit confused by the lib https://github.com/getbrevo/brevo-node/blob/584a95946ffd88a4874e2db85d35bff98bc54715/src/ApiClient.js#L598 does it mean that on every assign a new ApiClient will be initialized or it will be reused?